app/controllers/planned_runs_controller.rb

```diff
@@ -54,9 +57,24 @@ def update
 
     #Update all runs if Planned_run is changed
     Run.where(planned_run_id: @planned_run.id).find_each do |run|
-      run.date = @planned_run.date
-      run.training_plan = @planned_run.training_plan
-      run.save
+      update = true
+      if params[:checks]
+        params[:checks].each do |checks|
```

Comment:
Not 100% sure how this would work here. Since I have to loop through all of the 'checks' to see which ones not to update, how would this line work? If this can save a couple lines, I'm super interested to see how this works!! @epugh